web/assets/js/utils/crypto.ts

```diff
@@ -0,0 +1,9 @@
+import {entropyToMnemonic} from "@cosmjs/crypto/build/bip39";
+import {generateEntropy} from "@gnolang/tm2-js-client";
+
+/**
+ * Generates a random bip39 mnemonic
+ */
+export const generateMnemonic = (): string => {
+    return entropyToMnemonic(generateEntropy())
```

Comment:
Should it work on localhost? it returns ` randomFillSync is not a function`

Reply:
This is a really good catch, `randomFillSync` is a node method, and won't work on the browser.
I've substituted it for a browser-friendly alternative:
280d846157e895164ce9d469ccca7d10812f84e8